dist/src/main/dist/conf/client-hazelcast.xml

```diff
@@ -14,6 +14,22 @@
         <cluster-members>
             <!--MEMBERS-->
         </cluster-members>
+
+        <ssl enabled="true">
```

Comment:
Can this cause problems for test that rely on the client, but don't have HZ enterprise enabled??